iOS/Flows/Threads/ThreadCollectionView.swift

```diff
@@ -42,59 +42,3 @@ extension ThreadCollectionView: MessageSendingCollectionViewType {
         return .zero
     }
 }
-
-class ThreadTimeMachineCollectionViewLayout: MessagesTimeMachineCollectionViewLayout {
```

Comment:
No need to have a subclass for threads. We can configure the base class as needed.